Source/WebCore/css/parser/CSSPropertyParser.cpp

```diff
@@ -1228,11 +1228,11 @@ bool CSSPropertyParser::consumeBorderImage(CSSPropertyID property, bool importan
         };
         switch (property) {
         case CSSPropertyWebkitMaskBoxImage:
```

Comment:
`-webkit-mask-box-image` is utterly broken. It's implemented as a longhand, and is not parsed here in `consumeBorderImage()`. It's instead parsed in `consumeWebkitBorderImage()`.

So I think you should just remove this code. Or turn `-webkit-mask-box-image` into a shorthand.

Reply:
I will remove the code. A better implementation is a project for another day.